internal/checker/terraform/rules/ibm/classic_infrastructure/tfibm_user_has_api_key_enabled.rego

```diff
@@ -0,0 +1,58 @@
+# METADATA
+# title: "Verify user has api key enabled"
+# description: "This policy ensures robust protection of sensitive information by enforcing security measures for passwords. It verifies that passwords adhere to specific requirements, including minimum length, complexity, and absence of hardcoded passwords."
+# scope: package
+# related_resources:
+# - https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/compute_user
+# custom:
+#   id: CB_TFIBM_049
+#   severity: HIGH
+package lib.terraform.CB_TFIBM_049
+
+import rego.v1
+
+isvalid(block) if {
+	block.Type == "resource"
+	some label in block.Labels
+	supported_resource := ["ibm_compute_user", "joe"]
+	label in supported_resource
+}
+
+resource contains resource if {
+	some block in pass
+	resource := concat(".", block.Labels)
+}
+
+resource contains resource if {
+	some block in fail
+	resource := concat(".", block.Labels)
+}
+
+pass contains resource if {
```

Comment:
Based on the [resource](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/compute_user):

`has_api_key- (Optional, Bool) ...`

but the attribute indicates on a pass block instead on a fail block.